### ISSUE TITLE
"Unhandled Error in Deferred"

### ISSUE THREAD

**Issue by rbuchmueller** [fhamborg/news-please#46]
Hi,
i am working with Python 3.6.3 and pip 9.0.1 on Ubuntu 16.04.3 LTS.
news-please is not working right after the installation. 

The installation process was absolved successfully.
After starting "news-please" in the cli an "Unhandled error in Deffered" pops up.
In python programms "newsplease" can be imported, but not used.
The call news-please had to be made with sudo, else there appears an Permission Error.

You can see the stacktrace below.

Cheers,
Raphael

`raphael@raphael-Latitude-E6330:~$ sudo news-please
[sudo] password for raphael: 
[newsplease.config:165|INFO] Loading config-file (/home/raphael/news-please-repo/config/config.cfg)
[newsplease.config:165|INFO] Loading config-file (/home/raphael/news-please-repo/config/config.cfg)
[__main__:253|INFO] Removed /home/raphael/news-please-repo/.resume_jobdir/f03a98d15778ac99eeb8c578aa8c224b since '--resume' was not passed to initial.py or this crawler was daemonized.
[newsplease.config:165|INFO] Loading config-file (/home/raphael/news-please-repo/config/config.cfg)
[newsplease.config:165|INFO] Loading config-file (/home/raphael/news-please-repo/config/config.cfg)
Unhandled error in Deferred:

[__main__:253|INFO] Removed /home/raphael/news-please-repo/.resume_jobdir/861e0b7ca3034017282d27dce656d520 since '--resume' was not passed to initial.py or this crawler was daemonized.
[__main__:253|INFO] Removed /home/raphael/news-please-repo/.resume_jobdir/5011d55eaa1b745eefb709134271e173 since '--resume' was not passed to initial.py or this crawler was daemonized.
Unhandled error in Deferred:

Unhandled error in Deferred:

[newsplease.__main__:270|INFO] Graceful stop called manually. Shutting down.
`

**Comment by fhamborg** [comment]
> In python programms "newsplease" can be imported, but not used.
What specifically happens if you import news-please, and call one of the functions provided by the class NewsPlease?

**Comment by rbuchmueller** [comment]
an import error because news-please cannot be found while running with python.

I've changed my Python to 3.5, and after #47  it works.
